### ISSUE TITLE
Need to update deps/build.jl with the correct hash code for sentiment.tar.gz

### ISSUE THREAD

**Issue by AsafManela** [JuliaText/TextAnalysis.jl#82]
From travis-ci:
```
Info: Calculated hash 5dcb031eccf01bb0b2d074281140679683f73603a54caa79941a1df1c8a6d70d for file /home/travis/.julia/v0.6/TextAnalysis/deps/usr/downloads/sentiment.tar.gz
============================[ ERROR: TextAnalysis ]=============================
LoadError: Hash Mismatch!
  Expected sha256:   f237378f3f866c7e697ed893b4208878a6c5dd111eddcebc84ac55dab3885004
  Calculated sha256: 5dcb031eccf01bb0b2d074281140679683f73603a54caa79941a1df1c8a6d70d
while loading /home/travis/.julia/v0.6/TextAnalysis/deps/build.jl, in expression starting on line 44
================================================================================
```

**Comment by AsafManela** [comment]
Fixed by 26e94ff

**Comment by aviks** [comment]
Sorry for the faulty release!
